### DISCUSSION TITLE
Param in partial path segment not possible in v6?

### DISCUSSION THREAD

**Discussion by dsreitan** [remix-run/react-router#10889] [v5 to v6 Migration] [answered]
In our app we have routes like `/view--:id`, but after attempting to upgrade - we can't hit these routes anymore. 

Is this a breaking change in v6? Or can we work around it somehow?

**Comment by kiliman** [comment, marked as answer]
Yes, RR v6 does not support partial params. You'll need to create a route like `/:view` and in your loader, strip off the leading `"view--"` to get the id.
